AlpsSDK/MatchMonitor.swift

```diff
@@ -54,12 +54,15 @@ public class MatchMonitor: RemoteNotificationManagerDelegate {
         guard let deviceId = device.id else { return }
         MatchesAPI.getMatches(deviceId: deviceId) { (matches, error) in
             guard let matches = matches, matches.count > 0, error == nil else { return }
-            self.deliveredMatches = Set(matches)
-            self.delegate?.didFind(matches: matches, for: device)
+            let union = self.deliveredMatches.union(Set(matches))
+            if union != self.deliveredMatches {
+                self.deliveredMatches = union
+                self.delegate?.didFind(matches: matches, for: device)
```

Comment:
great, so now if matches are already delivered they won't be a second time right ?